plugins/hls-refactor-plugin/test/Main.hs

```diff
@@ -1669,8 +1669,10 @@ suggestImportTests = testGroup "suggest import actions"
     , test True []          "f = empty"                   []                "import Control.Applicative (empty)"
     , test True []          "f = empty"                   []                "import Control.Applicative"
     , test True []          "f = (&)"                     []                "import Data.Function ((&))"
-    , test True []          "f = NE.nonEmpty"             []                "import qualified Data.List.NonEmpty as NE"
-    , test True []          "f = Data.List.NonEmpty.nonEmpty" []            "import qualified Data.List.NonEmpty"
+    , ignoreForGHC94 "On GHC 9.4 the error message doesn't contain the qualified module name"
+      $ test True []          "f = NE.nonEmpty"             []                "import qualified Data.List.NonEmpty as NE"
+    , ignoreForGHC94 "On GHC 9.4 the error message doesn't contain the qualified module name"
+      $ test True []          "f = Data.List.NonEmpty.nonEmpty" []            "import qualified Data.List.NonEmpty"
```

Comment:
This seems like rather important regressions, we ought to create an issue for that

Reply:
I opened https://gitlab.haskell.org/ghc/ghc/-/issues/22130 a while ago but I didn't realise that it also affected import suggestions. Unfortunately it seems difficult to fix this in HLS and we have to wait for a fix in GHC.

Reply:
I've updated the message to point to this ticket.